src/ray/raylet/lineage_cache.cc

```diff
@@ -339,7 +339,7 @@ void LineageCache::EvictTask(const TaskID &task_id) {
   }
 
   // Evict the task.
-  RAY_LOG(DEBUG) << "Evicting task " << task_id << " on " << client_id_;
+  RAY_LOG(DEBUG) << "Evicting task " << task_id;
```

Comment:
`on node_id` is useful. We can get local node id from the gcs client.

Reply:
done